.github/workflows/ci.yml

```diff
@@ -0,0 +1,206 @@
+name: CI
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: '*'
+
+env:
+  PIP_DISABLE_PIP_VERSION_CHECK: 1
+  PYTHONUNBUFFERED: 1
+  PYTHONIOENCODING: utf-8
+
+defaults:
+  run:
+    shell: bash -l {0}
+
+jobs:
+  build:
+    name: build
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out
+      uses: actions/checkout@v2
+    - name: Set up python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
+    - name: Set up node
+      uses: actions/setup-node@v2
+      with:
+        node: 14.x
+    - name: Install packaging dependencies
+      run: |
+        set -eux
+        python -m pip install -vv -U --user pip wheel setuptools
+        yarn --version || npm install -g yarn
+    - name: Pre-install node dependencies
+      run: |
+        set -eux
+        cd js
+        yarn --ignore-optional
+    - name: Build sdist
+      run: python setup.py sdist
+    - name: Build wheel
+      run: python setup.py bdist_wheel
+    - name: Collect and hash distributions
+      run: |
+        set -eux
+        cp js/lab-dist/jupyter-threejs-*.tgz dist
+        cd dist
+        sha256sum * | tee SHA256SUMS
+    - name: Upload distributions
+      uses: actions/upload-artifact@v2
+      with:
+        name: dist ${{ github.run_number }}
+        path: ./dist
+
+  docs:
+    name: docs
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out
+        uses: actions/checkout@v2
+      - name: Install apt dependencies
+        run: |
+          set -eux
+          sudo apt install pandoc
+      - name: Set up python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+      - name: Install packaging dependencies
+        run: |
+          set -eux
+          python -m pip install -vv -U --user pip wheel setuptools
+      - name: Set up node
+        uses: actions/setup-node@v2
+        with:
+          node: 14.x
+      - name: Install package and docs dependencies
+        run: |
+          set -eux
+          pip install -vv -U .[docs,examples,test] requests_cache 'traitlets==4.*'
```

Comment:
yeah, lotta force-pushin', hard to dig back up. It was something about one of the fancy enum traits...

Reply:
Made #351, got CI to [show the issue](https://github.com/bollwyvl/pythreejs/runs/1944296845?check_suite_focus=true#step:9:37).

Given there is _no_ functional CI working, it would be lovely to get this merged so we're not having to look at a fork... likely we could iterate more effectively, and in parallel...